tests/yam/validate/validate_migration_logs.pm

```diff
@@ -21,10 +21,16 @@ sub run {
     script_run("tar czvf /tmp/udev_rules.tar.gz /etc/udev/rules.d/*", {timeout => 60});
     upload_logs("/tmp/udev_rules.tar.gz", failok => 1);
 
+    # fatal migration errors that lead to failed test run
```

Comment:
to shorten the comment (as it is not really necessary to say 'that lead to failed test run'  due those are technicalities from openQA, there is a clear `die` and it is just better to reflect functional code for testing), something like this could be more straightforward:
`# major errors` here and `#minor errors below` could work and being less verbose.